### DISCUSSION TITLE
Adding back beanio to Apache Camel project

### DISCUSSION THREAD

**Discussion by davsclaus** [beanio/beanio#156] [Show and tell]
The Apache Camel project have had support for beanio since 2012 in the Camel 2.10 release onwards. However a few years ago when beanio was dying and we had to cleanup unmaintained modules, then camel-beanio was removed in Camel 3.17 onwards.

However we can now see that this project with beanio v3 is active and that we are bringing back camel-beanio in our upcoming Camel 4.4 release
https://issues.apache.org/jira/browse/CAMEL-20369

That said it would be good to know if this project is on the way for a 3.0 GA release, as the last release is M3. Perhaps this project would consider releasing what you have as-is as 3.0 GA to build more confidence users can use this.

**Comment by bjansen** [comment]
That's great news for both Camel and BeanIO!
I just released 3.0.0, the last milestone was stable enough.

**Reply by davsclaus** [reply]
Thanks a lot, we have upgraded to 3.0.0 for our 4.4 release.
